_generated/HelmDeployV0/src/helm.ts

```diff
@@ -84,7 +84,11 @@ async function run() {
     var kubectlCli: kubernetescli;
     var externalAuth = connectionType === "None" && (command === "install" || command === "upgrade");
     if (externalAuth && !tl.getVariable("KUBECONFIG")) {
-        tl.error("KUBECONFIG kube configuration file path must be set when connectionType is none and command is install or upgrade.");
+        if (fs.existsSync("$HOME/.kube/config")) {
+            tl.setVariable("KUBECONFIG", "$HOME/.kube/config");
```

Comment:
Not blocking, but I still have a question about how this variable reportedly worked in 0.223.1 without KUBECONFIG set